bot/app/__init__.py

```diff
@@ -44,25 +44,33 @@ def start(message):
 	user = db_objects.User(u_id, name, is_bot)
 
 	if not user.user_exists(u_id):
-		user.add_user(u_id, name, is_bot)
+		user.add_user()
 
-	bot.send_message(chat_id=u_id, text=render_template('start.html', name=name), \
-		parse_mode='html')
-
-@bot.message_handler(commands=['test'])
-def test(message):
-	with open(os.path.join(template_dir, 'test.html'), 'r') as f:
-		reply = f.read()
-	bot.send_message(chat_id=message.chat.id, text=reply, parse_mode='html')
+	with app.app_context():
```

Comment:
Why do we need the app context here?

Reply:
Weird question, that I don't fully understand myself I have to admit. It is a workaround that forces Flask's context object. Without forcing we get context errors for the `render_template()` functions. Never had this in 'pure' Flask apps and recon that this has something to do with using both the Flask-stuff inside the pyTelegramBotApi functions.